src/features/postSlice.js

```diff
@@ -1,10 +1,13 @@
 import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import axios from "axios";
```

Comment:
i don't think you are using axios here as all the api calls are in the services folder

Reply:
Ok i will remove it in next PR.